core/context.go

```diff
@@ -85,6 +85,47 @@ func (c *Context) FilterUsers(filters ...string) (filtered []User) {
 	return filtered
 }
 
+// GetRootUser gets the root user
+func (c *Context) GetRootUser() User {
+	var root User
+
+	for _, user := range c.users {
+		if user.IsRoot {
+			root = user
+			break
+		}
+	}
+
+	return root
+}
```

Comment:
I *think* this is equivalent:

```suggestion
// GetRootUser gets the root user
func (c *Context) GetRootUser() User {
	for _, user := range c.users {
		if user.IsRoot {
			return user
		}
	}

	return User{}
}
```